build-api/api.py

```diff
@@ -0,0 +1,10 @@
+from flask import Flask
+
+app = Flask(__name__)
+
+@app.route("/build")
+def build():
+    return "Hello World!"
+
```

Comment:
so right now this doesn't do anything. Do we need this to do something before we pull this in?